app/src/main/java/io/intrepid/contest/screens/contestjudging/scoresubmission/SubmissionEntriesPresenter.java

```diff
@@ -0,0 +1,29 @@
+package io.intrepid.contest.screens.contestjudging.scoresubmission;
+
+import android.support.annotation.NonNull;
+
+import java.util.List;
+
+import io.intrepid.contest.base.BasePresenter;
+import io.intrepid.contest.base.PresenterConfiguration;
+import io.intrepid.contest.models.Entry;
+
+import static io.intrepid.contest.screens.contestjudging.scoresubmission.SubmissionEntriesContract.View;
+
+class SubmissionEntriesPresenter extends BasePresenter<SubmissionEntriesContract.View>
+        implements SubmissionEntriesContract.Presenter {
+
+    private final List<Entry> entries;
+
+    SubmissionEntriesPresenter(@NonNull View view,
+                               @NonNull PresenterConfiguration configuration, List<Entry> entries) {
```

Comment:
`List<Entry> entries` on new line. Also should annotate it as `@NonNull` or perform a null check since at some point you pass it to the adapter which calls `addAll`. This will cause a crash if entries is null.